apps/src/StudioApp.js

```diff
@@ -1724,7 +1724,7 @@ StudioApp.prototype.setConfigValues_ = function (config) {
   this.requiredBlocks_ = config.level.requiredBlocks || [];
   this.recommendedBlocks_ = config.level.recommendedBlocks || [];
 
-  if (config.ignoreLastAttempt) {
+  if (config.ignoreLastAttempt || config.hasContainedLevels) {
```

Comment:
Might be worth a comment on why hasContainedLevels leads to us ignoring last attempt.